CurrentSensing.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <assert.h>
+#include "CurrentSensing.h"
+
+void ConversionForCurrent(int index, float adcValues[], float minRange, float maxRange, int currentSamples[], int maxDigitalValue){
+	int adcRange = maxRange - minRange;
+
+	if(adcValues[index] < maxDigitalValue){
+		currentSamples[index] = round(adcRange * adcValues[index]/(maxDigitalValue-1));
+	}
+}
+
+void checkMinRange(int index, float adcValues[], float minRange, float maxRange, int currentSamples[], int maxDigitalValue){

```

Comment:
Name the function with its purpose rather than implementation.
Checking the min range is the implementation. Its purpose is to adjust to the offset.